### DISCUSSION TITLE
LAPRASについて「サポートに聞くほどではないんだけど困っていること、疑問に思っていること」を教えてください

### DISCUSSION THREAD

**Discussion by UserSuccessTeamLAPRAS** [lapras-inc/lapras-user-community#5] [General]
いつもLAPRASをご利用いただきありがとうございます 🙏 

LAPRASについて
「サポートに聞くほどではないんだけど実は困っていること、疑問に思っていること」
はありませんか？

ちょっとした疑問でも構いません、ぜひ教えてください！

**Comment by lamrongol** [comment]
スカウトメールはLAPRAS専用のメールアドレス(<企業名>@mail.lapras.com)から送信されてると思いますが、これに普通に返信しても企業に通知される仕様になっているでしょうか？

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASサポートです。
スカウトメールにつきましては、企業専用のメールアドレス宛にそのまま返信いただければ企業に通知されます。

今後ともLAPRASをよろしくお願いします。

**Comment by genpsp** [comment]
メッセージ機能のアーカイブや、非表示機能が搭載される予定はありますでしょうか。

**Reply by UserSuccessTeamLAPRAS** [reply]
LAPRASユーザーサポートです。

メッセージ機能のアーカイブや、非表示機能につきましては、直近リリース予定はありません。
改善のご要望として開発チームに共有させていただきます。

今後ともLAPRASをよろしくお願いします。
